wasm/package.json

```diff
@@ -11,8 +11,9 @@
     "create_sc": "node massa_tools/include_base64.js assembly/create_sc.ts && asc assembly/create_sc.m.ts --target release --exportRuntime --textFile build/create_sc.wat --binaryFile build/create_sc.wasm && rm assembly/create_sc.m.ts",
     "caller": "asc assembly/caller.ts --target release --exportRuntime --textFile build/caller.wat --binaryFile build/caller.wasm",
     "local_hello_name_caller": "asc assembly/local_hello_name_caller.ts --target release --exportRuntime --textFile build/local_hello_name_caller.wat --binaryFile build/local_hello_name_caller.wasm",
+    "no_main": "asc assembly/no_main.ts --target release --exportRuntime --textFile build/no_main.wat --binaryFile build/no_main.wasm",
```

Comment:
I'm aware that this `package.json` file became rely on messy through time, but I don't know how to tackle https://github.com/massalabs/massa-sc-template/issues/3!

Reply:
I don't know... I'll never know.